test/optimizer/join_parquet_with_base.test

```diff
@@ -0,0 +1,415 @@
+# name: test/optimizer/join_parquet_with_base.test
```

Comment:
Great test.

You don't need to explicitly write the TPC-H queries, we have PRAGMA's for that. Take a look at `test/sql/tpch/tpch_sf01.test_slow`.

We have also stored all the answers to the TPC-H queries so you don't need to check the hash either, you can just compare it with a file.

Finally, please also change the extension of this to `test_slow`, as this will be a slow test due to dbgen.